frontend/www/js/omegaup/components/arena/Arena.vue

```diff
@@ -62,18 +67,28 @@
 
 <script lang="ts">
 import { Vue, Component, Prop, Emit } from 'vue-property-decorator';
+import { types } from '../../api_types';
 import T from '../../lang';
 import { Tab } from '../problem/Details.vue';
 
 @Component
 export default class Arena extends Vue {
   @Prop({ default: false }) shouldShowRuns!: boolean;
+  @Prop({ default: false }) isAdmin!: boolean;
+  @Prop({ default: () => [] }) clarifications!: types.Clarification[];
   @Prop() contestTitle!: string;
   @Prop() activeTab!: string;
   @Prop() backgroundClass!: string;
 
   T = T;
   selectedTab = this.activeTab;
+  clarificationsHasBeenRead = true;
```

Comment:
nit:

```suggestion
  clarificationsHaveBeenRead = true;
```

"clarifications" está en plural, así que hay que usar el verbo en plural.

Reply:
Ups, sorry. 

Se te ocurre un nombre más claro para esto? Es que aún no termina de convencerme del todo